src/core/Float.tsx

```diff
@@ -1,24 +1,31 @@
 import * as React from 'react'
 import { useFrame } from '@react-three/fiber'
 import mergeRefs from 'react-merge-refs'
+import * as THREE from 'three'
 
 export type FloatProps = JSX.IntrinsicElements['group'] & {
   speed?: number
   rotationIntensity?: number
   floatIntensity?: number
   children?: React.ReactNode
+  floatingRange?: [number | undefined | null, number | undefined | null]
```

Comment:
This is meesy. We can let it be `floatingRange?: [number?, number?]`

Reply:
ohh